src/SelectableBox/utils.js

```diff
@@ -0,0 +1,25 @@
+import Form from '../Form';
+
+// eslint-disable-next-line import/prefer-default-export
+export const getType = (component, type) => {
```

Comment:
nit: maybe rename to `getInputType`, just so it's a tiny bit clearer what "type" is referring to at first glance? 🤔

Reply:
Agreed, fixed now